linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_media_export/rule.yml

```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8
```

Comment:
`rhcos4` product selects this rule too.
This is such a broad rule that I wonder if it makes sense to have `prodtype` in this rule.

Reply:
I am not against, I changed it.